config/locales/en/wizards/claims/add_claim_wizard.yml

```
@@ -0,0 +1,51 @@
+en:
+  wizards:
+    claims:
+      add_claim_wizard:
+        provider_step:
+          continue: Continue
+          page_title: Accredited provider - %{contextual_text}
+          title: Accredited provider
+        mentor_step:
+          page_title: Mentors for %{provider_name} - %{contextual_text}
+          continue: Continue
+          label: Mentor
+          heading: Mentors for %{provider_name}
+          select_all_that_apply: Select all that apply
+        mentor_training_step:
+          page_title: Hours of training for %{mentor} - %{contextual_text} - %{provider_name}
+          hours_of_training_for_mentor: Hours of training for %{mentor}
+          hours_of_training: Hours of training
+          hours:
+            one: "%{count} hour"
+            other: "%{count} hours"
+          hours_hint:
+            full: The full amount of hours for standard training
+            remaining: The remaining amount of hours for standard training
+          other_amount: Another amount
+          custom_hours_completed_hint:
+            one: Enter whole numbers up to a maximum of %{count} hour
+            other: Enter whole numbers up to a maximum of %{count} hours
+          number_of_hours: Number of hours
+        check_your_answers_step:
+          page_title: Check your answers - %{contextual_text}
+          title: Check your answers
+          warning: You will not be able to change any of the claim details once you have submitted it.
```

```suggestion
          warning: You will not be able to change any of the claim details once you have submitted the claim.
```

Personal opinion but I think we should avoid using "it" (I know this is likely pre-existing copy)